README.md

```diff
@@ -1,16 +1,20 @@
 FirebaseUI: Simplyify Mobile Development with Firebase
 ======================================================
 
-FirebaseUI is a library that allows developers to quickly build native mobile applications with Firebase by binding common UI views to Firebase references or queries. Currently FirebaseUI supports `ListViews`, `GridViews`, and `RecyclerViews` on Android through custom Firebase Adapters and `UITableViews` and `UICollectionViews` on iOS through custom Data Sources.
+FirebaseUI is a library that provides common UI elements when building apps. This includes a 
+* basic Auth flow implementation on web, Android, and iOS
+* bindings for `ListViews`, `GridViews`, and `RecyclerViews` in Android through custom Firebase Adapters
+* bindings for `UITableViews` and `UICollectionVIews` on iOs through custom Data Sources.
 
 These views are populated with the appropriate `child_added`, `child_changed`, `child_removed`, and `child_moved` events as they occur at the Firebase reference with no additional developer overhead. Additionally, the Adapters and Data Sources can easily be subclassed to add additional functionality  with little additional overhead.
 
 FirebaseUI is built on top of [Firebase](https://www.firebase.com/?utm_source=firebaseui) for realtime data storage and synchronization. [Sign up](https://www.firebase.com/signup?utm_source=firebaseui) for a free account today!
 
 # FirebaseUI Clients
-FirebaseUI is available for both native mobile clients:
+FirebaseUI is available in:
```

Comment:
Can you change `in` ->`on`? I think it sounds better (FirebaseUI is available on $platform vs FirebaseUI is available in $platform)